### ISSUE TITLE
Use toggle-window by double-tapping "ctrl"

### ISSUE THREAD

**Issue by koooota** [Eugeny/tabby#5864]
Is it possible to switch the display by pressing the "ctrl" key twice like iterm2?
If you are currently unable to do so, do you plan to do so in the future?

**Comment by Eugeny** [comment]
No, multi-stroke shortcuts are not supported for the toggle-window shortcut, and it doesn't support "modifier-only" either - this is due to the toggle-window hotkey being handled by Electron as opposed to the others, which are handled by Tabby itself.
